src/main/java/ladder/controller/LadderGame.java

```diff
@@ -0,0 +1,44 @@
+package ladder.controller;
+
+import java.util.List;
+
+import ladder.domain.ladder.Ladder;
+import ladder.domain.line.ConnectionGenerator;
+import ladder.domain.player.Players;
+import ladder.view.LadderInputView;
+import ladder.view.LadderOutputView;
+
+public class LadderGame {
+
+    private final LadderInputView inputView;
+    private final LadderOutputView outputView;
+    private final ConnectionGenerator connectionGenerator;
+
+    public LadderGame(
+            final LadderInputView inputView,
+            final LadderOutputView outputView,
+            final ConnectionGenerator connectionGenerator
+    ) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.connectionGenerator = connectionGenerator;
+    }
+
+    public void run() {
+        try {
+            final List<String> playerNames = inputView.readPlayerNames();
+            final Players players = Players.from(playerNames);
```

Comment:
아래처럼 간소화해볼 수 있을 것 같습니다.

```suggestion
            final Players players = Players.from(inputView.readPlayerNames());